src/commands.js

```diff
@@ -0,0 +1,58 @@
+const log4js = require('log4js');
+const logger = log4js.getLogger('commands.js');
+
+const { parseFile, writeFile } = require ('./fileio.js');
+
+function processCommand(input, accounts) {
+    if (input == 'Quit') {

```

Comment:
This `if`, `startsWith` and `slice` approach is a little tough to read; I suspect using `input.split()` would give tidier results that won't break if you rename commands 😉

It'll also allow you to commonise more of the code between branches, which in turn probably allows refactoring where you just pick the function to use out of a dictionary.